client/platform/desktop/backend/native/viame.ts

```diff
@@ -249,7 +259,103 @@ async function train(
   return jobBase;
 }
 
+function checkMedia(
```

Comment:
No problem, I think this might of been a holdover from when I was initially coding and didn't have the 'converting...' stuff implemented and thought that displaying should be done immediately.